mysql-test/t/percona_userstat_bug728082.test

```diff
@@ -0,0 +1,43 @@
+--source include/have_innodb.inc
+--source include/count_sessions.inc
+
+SET @userstat_old= @@userstat;
+
+CREATE USER mysqltest_1@localhost;
+CREATE USER mysqltest_2@localhost;
+
+SET GLOBAL userstat=ON;
+
+connect (conn1,localhost,mysqltest_1,,);
+connection default;
+
+SELECT CONCURRENT_CONNECTIONS FROM INFORMATION_SCHEMA.USER_STATISTICS WHERE USER LIKE 'mysqltest_1';
+
+connect (conn2,localhost,mysqltest_1,,);
+connection default;
+
+--echo check inc2
+SELECT CONCURRENT_CONNECTIONS FROM INFORMATION_SCHEMA.USER_STATISTICS WHERE USER LIKE 'mysqltest_1';
+
+disconnect conn2;
```

Comment:
Disconnect will complete asynchronously on the server after the network connection is closed, this is a race condition with the statement below. You can sync by Threads_connected status var, and there are several ways to do it, easiest is probably to replace the global --source include/count_sessions.inc / wait_until_count_sessions.inc (which is BTW missed here) with a few local ones as needed